docker-lite/nginx/confd/nginx-config.toml

```diff
@@ -1,6 +1,6 @@
 [template]
 src = "nginx-config.conf.tmpl"
-dest = "/etc/bitwarden/nginx/server.conf"
+dest = "/etc/nginx/bitwarden.conf"
```

Comment:
didnt seem necessary to persist this to the mapped volume since it is auto-generated each launch of the container